src/styles/body.scss

```diff
@@ -0,0 +1,9 @@
+
+body {
+  margin: 0;
+  height: 100vh;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `height` property should be ordered alphabetically among other properties within the selector.